MyOnboarding/test/MyOnboardingApp.Contracts.Tests/Validation/ItemWithErrorsTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using MyOnboardingApp.Contracts.Errors;
+using MyOnboardingApp.Contracts.Models;
+using MyOnboardingApp.Contracts.Validation;
+using MyOnboardingApp.TestUtils.Extensions;
+using MyOnboardingApp.TestUtils.Factories;
+using NUnit.Framework;
+
+namespace MyOnboardingApp.Contracts.Tests.Validation
+{
+    [TestFixture]
+    public class ItemWithErrorsTests
+    {
+        [Test]
+        public void Create_ItemAndErrorCollectionSpecified_ReturnsInvalidatedItem()
+        {
+            var error = new Error(ErrorCode.DataValidationError, "Text is empty.", "text");
+            var errors = new [] { error };
+            var (testItem, _, expectedResult) = ItemVariantsFactory
+                .CreateItemVariants(
+                    new Guid("00000000-0000-0000-0000-000000000001"),
+                    string.Empty,
+                    new DateTime(2000, 01, 01),
+                    errors
+                );
+
+            var result = ItemWithErrors.Create(testItem, errors);
+
+            Assert.Multiple(() =>
+            {
+                Assert.That(result.WasOperationSuccessful, Is.False);
+                Assert.That(result, Is.EqualTo(expectedResult).UsingItemWithErrorsEqualityComparer());
+            });
+        }
+
+
+        [Test]
+        public void Create_ItemAndEmptyErrorCollectionSpecified_ReturnsValidatedItem()
+        {
+            var errors = new Error[] {};
```

Comment:
I mislead you in https://github.com/petrareb/kentico-onboarding-cs/pull/4#discussion_r263750196 a bit, I'm sorry about that 😞. You should use the following code in this very case:
```suggestion
            var errors = Array.Empty<Error>();
```
(for the same reasons, you should use `Enumerable.Empty<T>()` in similar cases - just as described in https://github.com/petrareb/kentico-onboarding-cs/pull/4#discussion_r249161333)